tests/unit/test_notebooks.py

```diff
@@ -28,7 +28,8 @@
 SESSION_PATH = "examples/getting-started-session-based/"
 
 
-@pytest.mark.skipif(find_spec("cudf") is None, reason="needs cudf")
+# @pytest.mark.skipif(find_spec("cudf") is None, reason="needs cudf")
+@pytest.mark.skip(reason="there is a testbook version of this test")
 def test_session(tmpdir):
```

Comment:
@jperez999 since I am replacing this test with the testbook version I added `mark.skip` into the existing test, but we can remove it as well. what do u think? thanks.